### DISCUSSION TITLE
Satellite imagery

### DISCUSSION THREAD

**Discussion by santod2020** [weather-gov/api#687] [Ideas]
Can there be an API for satellite imagery? I have been scraping satellite imagery for an application, but something has changed recently, and I can't get the images in a timely manner. Sometimes they take up to 20 minutes for my raspberry pi to display them. Can there be a source for developers like it was early in the fall, or an api offered for satellite images?

**Comment by dwhitemv25** [comment]
What satellite imagery are you looking for?

**Comment by santod2020** [comment]
Anything from goes 16, although it was true for 18 too, on this web site: https://www.star.nesdis.noaa.gov/GOES/conus.php?sat=G16.... the images were being scraped so easily a couple of months ago.Thanks for any help or explanations you can offer.On Dec 23, 2023 8:37 PM, dwhitemv25 ***@***.***> wrote:
What satellite imagery are you looking for?

**Reply by dwhitemv25** [reply]
For the STAR GOES Imagery site, if you follow the "Download images" link for the channel(s) you're interested in on the image index page, you'll be taken to a directory index where the images are stored. Note the images that don't have the timestamp on them; those are the "current" versions for those formats. 

Is that what you were looking for? I don't know what the "previous" interface looked like, sorry.
